grafana-plugin/src/containers/UserSettings/parts/index.tsx

```diff
@@ -140,7 +141,9 @@ export const TabsContent = observer(({ id, activeTab, onTabChange, isDesktopOrLa
         ) : (
           <PhoneVerification userPk={id} />
         ))}
-      {activeTab === UserSettingsTab.MobileAppConnection && <MobileAppConnection userPk={id} />}
+      {!isUseProfileExtensionPointEnabled() && activeTab === UserSettingsTab.MobileAppConnection && (
+        <MobileAppConnection userPk={id} />
+      )}
```

Comment:
@Matvey-Kuk brought up a good point that we _might_ still want to show this tab on Grafana >= 10.3, but instead of showing the QR code content, show a message w/ a link pointing users to the new tab on the `/profile` page (you should be able to deep link to the "IRM" tab using `/profile?tab=irm`)